spec/core_functions/math/round.hrx

```diff
@@ -48,14 +48,18 @@ a {
 
 <===>
 ================================================================================
+<===> up/within_precision/options.yml
+:ignore_for:
+  - libsass
+
 <===> up/within_precision/input.scss
 // This is the smallest number that's representable as a float and in the
-// precision range to be considered equal to 5.
-a {b: round(0.4999999999900001)}
+// precision range to be considered equal to 0.5.
+a {b: round(0.499999999995)}
 
 <===> up/within_precision/output.css
 a {
-  b: 1;
+  b: 0;
```

Comment:
This is not consistent with the test name, which says it would round *up*

Reply:
Good call; this test is now redundant with `down/within_precision`, so I'll remove it.